src/main/java/com/bookstore/controller/OrderController.java

```diff
@@ -53,8 +58,9 @@ public OrderItemResponseDto getOrderItem(@PathVariable Long orderId,
     @Operation(summary = "Place an order",
             description = "Create a new order")
     public OrderResponseDto createOrder(
-            @RequestBody @Valid CreateOrderRequestDto requestDto) {
-        return orderService.save(requestDto);
+            @RequestBody @Valid CreateOrderRequestDto requestDto, Authentication authentication) {
+        User user = (User) authentication.getPrincipal();
+        return orderService.save(requestDto, user.getId());
```

Comment:
You can do transformation logic in the service. 
```suggestion
        return orderService.save(requestDto, authentication);
```